specification/_global/exists_source/SourceExistsRequest.ts

```diff
@@ -29,24 +29,65 @@ import {
 import { SourceConfigParam } from '@global/search/_types/SourceFilter'
 
 /**
+ * Checks if a document is stored in the `source`.
```

Comment:
I would word this slightly differently:
```suggestion
 * Checks if a document's `_source` is stored.
```